app/javascript/src/apis/contact.js

```diff
@@ -0,0 +1,39 @@
+const fetch = async () => {
+  const userExample1 = {
+    name: "Ronald Richards",
+    role: "Owner",
+    email: "richards@example.com",
+    profile_pic: "https://i.pravatar.cc/300",
+    created_at: new Date().toString(),
+  };
+  const userExample2 = {
+    name: "Jacob Jones",
+    role: "Owner",
+    email: "jones@example.com",
+    profile_pic: "",
+    created_at: new Date().toString(),
+  };
+  const mock = [];
+  let count = 0;
+  for (let i = 0; i < 50; i++) {
```

Comment:
Don't use for loops. Use HOFs. You can use relevant Array helpers. Example: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
Thanks, Yedhin for pointing this out. I have used the HOF function in place of the above code.